_maps/RandomRuins/SpaceRuins/infested_frigate.dmm

```diff
@@ -2476,9 +2458,7 @@
 /obj/effect/turf_decal{
 	dir = 1
 	},
-/obj/effect/turf_decal{
-	dir = 2
-	},
+/obj/effect/turf_decal,
```

Comment:
wtf

Reply:
why does this map have base turf decal type

Reply:
you could spin up a quick updatepaths like

```txt
/obj/effect/turf_decal : @DELETE
```

and then maplint against this or something. should be done in another PR though